backend/db/emails.py

```diff
@@ -0,0 +1,12 @@
+from sqlmodel import SQLModel, Field
+from datetime import datetime
+
+class Email(SQLModel, table=True):
+    __tablename__ = 'emails'
+    email_id: int = Field(default=None, primary_key=True)
+    email_subject: str
+    from_email: str
+    company_id: int = Field(foreign_key="company.id")  
```

Comment:
```suggestion
    company_id: int = Field(foreign_key="companies.company_id")
```
You might want to work with @m-banthwan on this, or check the PR